File: DSA Practice/LeetCode/2740. Find the Value of the Partition/solution.java
class Solution {
    public int findValueOfPartition(int[] nums) {
        Arrays.sort(nums);
        int res = nums[1] - nums[0];
        int n = nums.length;
        for (int i = 2; i < n; i++)
            res = Math.min(res, nums[i] - nums[i - 1]);
        return res;
    }
}

File: DSA Practice/LeetCode/2740. Find the Value of the Partition/solution.kt
class Solution {
    fun findValueOfPartition(nums: IntArray): Int {
        Arrays.sort(nums)
        var res = nums.get(1) - nums.get(0)
        val n = nums.size

        for (i in 2..n-1){
            res = Math.min(res, nums.get(i) - nums.get(i-1))
        }

        return res

    }
}